components/brave_ads/browser/notification_helper_android.cc

```diff
@@ -28,6 +29,11 @@ bool NotificationHelperAndroid::IsNotificationsAvailable() const {
 // Starting in Android 8.0 (API level 26), all notifications must be
 // assigned to a channel.
 bool NotificationHelperAndroid::IsBraveAdsChannelEnabled() const {
+  char osVersion[PROP_VALUE_MAX+1];
+  int os_version = __system_property_get("ro.build.version.release", osVersion);
```

Comment:
`__system_property_get` returns the string length of the value. value is set to system property

Reply:
How would you write it then?

Reply:
See above as I have provided code (which I have not built but will give you the gist)